src/GLPIMailer.php

```diff
@@ -186,10 +186,12 @@ public function getEmail(): Email
      * @param string|null $debug_header_line
      *
      * @return void
+     *
+     * @deprecated 10.1.0
```

Comment:
Since this method has been added only in main branch, I guess it's quite safe to drop it reather than add a depreciation